pipeline/relation_embedder/relation_embedder/bash_secrets_extension.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+set -euo pipefail
+
+OWN_FILENAME="$(basename $0)"
```

Comment:
For use in other container images, this should be in a shared location eventually.